docs/xml-sitemaps.md

```diff
@@ -0,0 +1,54 @@
+# XML Sitemaps
+
+XML Sitemaps are a means of providing key content and additional meta data around the frequency of updates on your website to search engines.
+
+The functionality for this component is provided by the [MSM Sitemap](https://github.com/humanmade/msm-sitemap) plugin for WordPress.
+
```

Comment:
What sitemaps will be generated? Like, is this Google News sitemaps, or something else?

Reply:
Added more info, these are just the basic type of sitemap - I've added #2 to track some improvements we could make out of the box.